_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,67 @@
+
+| Option | Type | Description |
+| :--- | :--- | :--- |
+| port | int | Represents the port that the `otel_logs_source` is running on. Default value is `21892`. |
+| path | string | Represents the path for sending unframed HTTP requests. You can use this option to support unframed gRPC with HTTP idiomatic path to a configurable path. The path should start with `/` and length should be at least 1. `/opentelemetry.proto.collector.logs.v1.LogsService/Export` endpoint is disabled for both gRPC and HTTP requests if the path is configured. The path can contain `${pipelineName}` placeholder, which is replaced with pipeline name.  If the value is empty and `unframed_requests` is `true`, then the path that the source provides is `/opentelemetry.proto.collector.logs.v1.LogsService/Export`. | 
+| request_timeout | int | Represents the request timeout duration in milliseconds. Default value is `10000`. |
+| health_check_service | Boolean | Enables the gRPC health check service under `grpc.health.v1/Health/Check`. Default value is `false`. |
+| proto_reflection_service | Boolean | Enables a reflection service for Protobuf services (see [ProtoReflectionService](https://grpc.github.io/grpc-java/javadoc/io/grpc/protobuf/services/ProtoReflectionService.html) and [gRPC reflection](https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md) documents). Default value is `false`. |
+| unframed_requests | Boolean | Enables requests that are not framed using the gRPC wire protocol. Default value is `false`. |
+| thread_count  | int | The number of threads to keep in the `ScheduledThreadPool`. Default value is `500`. |
+| max_connection_count | int | The maximum number of open connections allowed. Default value is `500`. |
+
+### SSL
+
+You can configure SSL in the `otel_logs_source` source with the following options.
+
+| Option | Type | Description |
+| :--- | :--- | :--- |
+| ssl | Boolean | Enables TLS/SSL. Default value is `true`. |
+| sslKeyCertChainFile | string | Represents the SSL certificate chain file path or Amazon Simple Storage Service (Amazon S3) path. For example, see the Amazon S3 path `s3://<bucketName>/<path>`. Required if `ssl` is set to `true`.  |
+| sslKeyFile | string | Represents the SSL key file path or Amazon S3 path. For example, see the Amazon S3 path: `s3://<bucketName>/<path>`. Required if `ssl` is set to `true`.  |
+| useAcmCertForSSL | Boolean | Enables TLS/SSL using certificate and private key from AWS Certificate Manager (ACM). Default value is `false`. |
+| acmCertificateArn | string | Represents the ACM certificate ARN. ACM certificate take preference over Amazon S3 or local file system certificate.  Required if `useAcmCertForSSL` is set to `true`. |
+| awsRegion | string | Represents the AWS Region to use ACM or Amazon S3.  Required if `useAcmCertForSSL` is set to `true` or `sslKeyCertChainFile` and `sslKeyFile` is the Amazon S3 path. |
```

Comment:
"Represents the AWS Region used by ACM or Amazon S3"? Should the last instance of "and" be "or" ("or `sslKeyCertChainFile` or `sslKeyFile` is the Amazon S3 path")?

Reply:
Using "or" makes more sense here.